frontend/src/components/applyForm/utils.tsx

```diff
@@ -559,11 +560,16 @@ export const getRequiredProperties = (
 };
 
 export const isFieldRequired = (
-  definition: string,
+  definition: string | string[] | undefined,
   requiredFields: string[],
 ): boolean => {
-  const path = removePropertyPaths(definition);
-  return requiredFields.indexOf(path) > -1;
+  if (!definition) return false;
```

Comment:
when I wrote this I figured it was simple enough that it didn't need testing 😬 . I'm still good with this as is, but would be even happier with some tests at this point